HACKING.md

```diff
@@ -277,6 +117,74 @@ Or in order to log starting at page reload:
 
     > window.sessionStorage["debugging"] = "channel"
 
+## Building Cockpit binaries
+
+For more complex hacking on Cockpit beyond the user interface, you need to
+build the Cockpit binaries locally and install the relevant dependencies.
+Currently, recent x86_64 architectures of Fedora are most often used for
+development.
+
+Check `tools/cockpit.spec` for the concrete Fedora build dependencies.
+The following should work in a fresh Git clone:
+
+    $ sudo yum-builddep tools/cockpit.spec
```

Comment:
Not on RHEL, CentOS. Documentation should stick with yum for the time being. Renaming to dnf was a big fail from a usability perspective.